orangewidget/utils/combobox.py

```diff
@@ -13,6 +14,10 @@
 )
 
 
+# we want to have combo box maximally 25 characters wide
+MAXIMUM_CONTENTS_LENGTH = 25
```

Comment:
I would still leave this as a class constant/attribute so that it is at least possible to change it per instance in rare cases when that is needed (although setting it in the constructor is being deprecated).

Reply:
I put back the class attributes but still left this constant here - now attributes from both ComboBox, ComboBoxSearch get the default value from this constant. I also added setter for maximuContentLength to ComboBoxSearch that they work the same.